policyengine_us/parameters/gov/states/ri/tax/income/exemption/reduction/percentage.yaml

```
@@ -0,0 +1,41 @@
+description: Rhode Island phases this percentage of the total exemption amount based on modified agi.
+metadata:
+  type: single_amount
+  threshold_unit: /1
```

`/1` results in the parameter being formatted as a percentage in the app. This is a whole number.
```suggestion
  threshold_unit: 1
```